### DISCUSSION TITLE
Selecting jobs while in game via the new character tab has broken UI.

### DISCUSSION THREAD

**Discussion by Crazyjr20** [FakeFishGames/Barotrauma#15255] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [X] My issue happened while using mods.

### What happened?

Trying to change jobs while in game using the create new character function is impractical due to broken UI.
See attached images for vanilla and my own mod that adds new loadouts. This is more problematic for my mod as you cant see what loadouts you are selecting, and very problematic for mods that add more jobs as those are unselectable without returning to lobby.
![vanillauibroken](https://github.com/user-attachments/assets/0124d3a7-5bc7-49dd-9ac5-53efc4d7295e)
![moddedjobui](https://github.com/user-attachments/assets/7fbe52a5-f61d-4e67-8186-0456a80ca42d)
This can be kind of rectified by just quitting the server and rejoining to do it in lobby, but thats besides the point that this is still a bug that I don't think has been reported before so I'd assume you guys are unaware of it.

### Reproduction steps

1. Start a multiplayer campaign
2. Press tab and go Character.
3. Select "Create New" 
4. Try to change jobs to anything below the top row.

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Happens in both single player and multiplayer

### Version

v1.6.19.1 (Unto the Breach Update Hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by AziKarnage** [comment]
This issue is still plaguing Custom Gamemodes as well and has been for a while

**Comment by Regalis11** [comment]
Thank you for the report! This has been fixed in our private repo now, fix coming up in the next unstable build and the upcoming release.
